src/de.pengutronix.rauc.Installer.xml

```diff
@@ -36,6 +36,21 @@
       <arg name="compatible" type="s" direction="out" />
       <arg name="version" type="s" direction="out" />
     </method>
+ 
+    <!--
+    InfoBundle: D-Bus variant of rauc info <bundle>
+    @bundle: full path to the queried bundle.
+    @args: Additional arguments to pass
+    @compatible: the compatible string from the bundle
+    @version: the version string from the bundle
+   -->
+   <method name="InfoBundle">
+      <arg name="bundle" type="s" direction="in" />
```

Comment:
Perhaps we should rename the argument to `source`, as it has the same function as that argument in InstallBundle.

Reply:
I have no strong opinion on this, but yes it might be more consistent (at least within the set of non-deprecated API methods).